crates/wit-parser/src/resolve.rs

```diff
@@ -1474,22 +1490,26 @@ impl Remap {
     fn append(
         &mut self,
         resolve: &mut Resolve,
-        unresolved: UnresolvedPackage,
+        unresolved: &mut UnresolvedPackage,
```

Comment:
How come this changed from `UnresolvedPackage` to `&mut`? The previous signature helped reflect that ownership was transferred and I think it'd be best to keep it if possible.